src/test/java/model/TestLottoNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package model;
+
+import java.util.List;
+
+class TestLottoNumberGenerator implements view.util.LottoNumberGenerator {
```

Comment:
```suggestion
class TestLottoNumberGenerator implements LottoNumberGenerator {
```
패키지 import 하여도 상관없어보여요.